.rubocop.yml

```diff
@@ -13,7 +13,6 @@ AllCops:
   - pkg/**/*
   - spec/fixtures/**/*
   - vendor/**/*
-inherit_from: .rubocop_todo.yml
```

Comment:
Pretty sure this means that todo files are just ignored

Reply:
So for now the todo file is empty so this should be fine.
There is a ticket being added to the backlog that will allow us to make this configurable in the PDK.